### PR TITLE
move nlog code to seperate package

### PR DESCRIPTION
**Pull request by Jetski5822**
@jersiovic Yo I need this for a local project, whats your thoughts?

### CODE REVIEW

File: src/Orchard.Cms.Web/Startup.cs [hunk 1]
```diff
@@ -40,9 +38,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseStaticFiles();
             loggerFactory.AddConsole(Configuration);
```
Comment (jersiovic) [inline src/Orchard.Cms.Web/Startup.cs]: Only thing still to do is to remove `loggerFactory.AddConsole(Configuration); ` line and adding a console target to NLog config file. https://github.com/NLog/NLog/wiki/Console-target

Comment (Jetski5822) [inline src/Orchard.Cms.Web/Startup.cs]: Might as well create a Orchard.Logging.Console?

Comment (jersiovic) [inline src/Orchard.Cms.Web/Startup.cs]: It is only to add the new target in \src\Orchard.Cms.Web\NLog.config file. We don't need an extra module, do we?

File: src/OrchardCore/Orchard.Logging.NLog/ApplicationBuilderExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+using NLog.LayoutRenderers;
+using NLog.Web;
+
+namespace Orchard.Logging
+{
+    public static class ApplicationBuilderExtensions
+    {
+        public static IApplicationBuilder AddNLogWeb(this IApplicationBuilder app, ILoggerFactory loggerFactory)
+        {
+            LayoutRenderer.Register<TenantLayoutRenderer>(TenantLayoutRenderer.LayoutRendererName);
```
Comment (sebastienros) [inline src/OrchardCore/Orchard.Logging.NLog/ApplicationBuilderExtensions.cs]: I would have let this code in the application and just put the class in the project.

The same way you do `loggerFactory.AddConsole()` and don't call an extension method which is in the console package.

**Pull request merged**